v2/concordium/types.proto

```diff
@@ -499,6 +530,14 @@ message AccountInfo {
   // Canonical address of the account. This is derived from the first credential
   // that created the account.
   AccountAddress address = 10;
+  // The stake on the account that is in cooldown.
+  // There can be multiple amounts in cooldown that expire at different times.
+  repeated Cooldown cooldowns = 11;
```

Comment:
**Suggestion (non-blocking):** Could we mention that this field was introduced in P7 and how it behaves for protocols prior to that